modules/hardware/definition.nix

```diff
@@ -74,6 +74,41 @@
       };
     };
 
+    audio = {
+      # TODO? Should add AudioVM enabler here?
```

Comment:
IMO these comments can be removed. This file is for definitions, not enabling options.
You've correctly defined the option here with example. The only thing worth mentioning here with the example is that the whole IOMMU group must be defined to get passed through with current implementation. It's not obvious from the example but if one looks at the devices (`ISA Bridge @ 1f.0`, `SMBus @ 1f.4` and `Serial bus controller @ 1f.5` in addition the actual `Audio device @ 1f.3`). 
```
00:1f.0 ISA bridge: Intel Corporation Device 519d (rev 01)
00:1f.3 Audio device: Intel Corporation Device 51ca (rev 01)
00:1f.4 SMBus: Intel Corporation Alder Lake PCH-P SMBus Host Controller (rev 01)
00:1f.5 Serial bus controller: Intel Corporation Alder Lake-P PCH SPI Controller (rev 01)
```
My proposal for the comment is:
`With the current implementation, the whole PCI IOMMU group - 1f in the example from Lenovo X1 Carbon - must be defined for passthrough to AudioVM`

Reply:
Agree, fixed.